Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -224,3 +192,29 @@ private func cookieFrom(refTag: RefTag, project: Project) -> HTTPCookie? {
 
   return HTTPCookie(properties: properties)
 }
+
+private func fetchProjectAndLiveStreams(projectOrParam: Either<Project, Param>)
```

Comment:
Yes the fetch does take slightly longer, but only when coming from a Push or other Param-based route (since in that case the project does not emit immediately with the `prefix` value). I noticed the UI was stuttering a bit on swipe after merging Live stuff so wanted to cut down on the number of times the view is rendered.